draft/2020-06-23-this-week-in-rust.md

```diff
@@ -36,6 +36,24 @@ Check out [this week's *This Week in Rust Podcast*]()
 * [SIMD library plans](https://vorner.github.io/2020/05/08/simd-library-plans.html)
 * [Tips for Faster Rust Compile Times](https://endler.dev/2020/rust-compile-times/)
 * [Rust Analyzer Changelog #30](https://rust-analyzer.github.io/thisweek/2020/06/22/changelog-30.html)
+* [Building A Blockchain in Rust & Substrate: [A Step-by-Step Guide for Developers]](https://hackernoon.com/building-a-blockchain-in-rust-and-substrate-a-step-by-step-guide-for-developers-kc223ybp)
```

Comment:
Looks like you've got an extra bracket on this line (right after the word "Developers")